src/EasyList/ParseAdd.cs

```diff
@@ -1,10 +1,46 @@
 ﻿using EasyList.Enums;
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
 
 namespace EasyList
 {
+    public static class OptionParser
+    {
+        public static IEnumerable<(string option, string value)> ParseOptions(string[] options, string[] inputs)
```

Comment:
This function doesn't appear to be parsing anything. Looks like it's just pairing up the option name with the value for it?
I cannot tell. 
What's this function supposed to do? And where will it be used?
Where are `options` and `inputs` coming from?

Reply:
Theres a gif on discord and some code for it. Its supposed to replace the current input parser.

- you have a set amount of options  `options = {"-title", "-description"}`

- Then you get input from the user and do `.Split(" "); ` example users input: "-title clean my room -description its very messy"

- Pass the options and inputs down to the method

- what you get back is the option combined with its value so basically you have "-title", "clean my room" and ".description", "its very messy"